ci/build-centos.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
```

Comment:
What's the difference between build-centos.sh and build.sh files?

Reply:
As it was before. build.sh provide an interface to control testing and other activities, build-${platform} - specific scenarios to work with platforms. Original idea was to unite all build scenarios into 1 recipe, to avoid duplication.